bridge/evm/contracts/BridgeVault.sol

```diff
@@ -56,7 +56,8 @@ contract BridgeVault is Ownable, IBridgeVault {
         wETH.withdraw(amount);
 
         // Transfer the unwrapped ETH to the target address
-        recipientAddress.transfer(amount);
+        (bool success,) = recipientAddress.call{value: amount}("");
```

Comment:
how do we prevent reentrancy attack here?

Reply:
the root public function `transferBridgedTokensWithSignatures` is which is `nonReentrant`. All intermediate function are either private or guarded by `onlyOwner`. So it looks like right.

Is there any downside of adding `nonReentrant` to `transferETH` and `transferERC20` just to be safe?

Reply:
Gas mostly